Core/App/AppAPI.php

```diff
@@ -65,19 +67,22 @@ public function run(): bool
     }
 
     /**
-     * Returns true if the client is authenticated with the header token.
-     *
-     * @author Ángel Guzmán Maeso <angel@guzmanmaeso.com>
```

Comment:
@NeoRazorX the author should be mandatory, and it don't should be removed. Mostly if you are saying that you are AGPLv3 or similar